telegram/chat.py

```diff
@@ -270,6 +258,35 @@ def send_message(self, *args, **kwargs):
         """
         return self.bot.send_message(self.id, *args, **kwargs)
 
+    def send_media_group(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.send_media_group(Chat.id, *args, **kwargs)
+
+        Where Chat is the current instance.
```

Comment:
All older docstrings use `update.message.chat.id` instead of `Chat.id`. We should decide which one we want to use and stick with through the whole file.

Reply:
Thanks for the catch! I went for `update.*` as that's the most common use case and imo with that it's easier to understand what the shortcut does for newbies. However, I changed to `update.effective_chat.id` b/c that should be used in the manual way, too, imho.

Reply:
best way imo as well